### PR TITLE
BZ2056623: Deploy templates to default namespace

### PR DESCRIPTION
**Pull request by lmandavi**
This PR is associated with:
https://bugzilla.redhat.com/show_bug.cgi?id=2056623
https://issues.redhat.com/browse/CNV-16535

It applies to the CNV 4.10 release.

### CODE REVIEW

**Comment by gouyang** [comment]
LGTM

**Review by dominikholler — APPROVED** [review]
/lgtm

File: virt/virt-4-10-release-notes.adoc [hunk 1]
```diff
@@ -122,6 +122,12 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-10-known-issues"]
 == Known issues
 
+//Known issues at time of 4.10 release
+//BZ-2054650
+* The web console does not display templates that are deployed to a custom namespace. Only templates deployed to the default namespace will display in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2056623*])
++
+** As a workaround, avoid deploying templates to a custom namespace. To use templates from the web console, deploy templates to the default namespace.
```
Comment (sjhala-ccs) [inline virt/virt-4-10-release-notes.adoc:129]: Suggest removing the second sentence in the workaround description.

File: virt/virt-4-10-release-notes.adoc [hunk 2]
```diff
@@ -122,6 +122,12 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 [id="virt-4-10-known-issues"]
 == Known issues
 
+//Known issues at time of 4.10 release
+//BZ-2054650
+* The web console does not display templates that are deployed to a custom namespace. Only templates deployed to the default namespace will display in the web console. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2056623*])
```
Comment (sjhala-ccs) [inline virt/virt-4-10-release-notes.adoc:127]: Suggestion: s/templates/virtual machine templates for the first instance of templates.

File: virt/virt-4-10-release-notes.adoc (hunk 1 above)
Comment (lmandavi) [reply virt/virt-4-10-release-notes.adoc:129]: Updated the content.

File: virt/virt-4-10-release-notes.adoc (hunk 2 above)
Comment (lmandavi) [reply virt/virt-4-10-release-notes.adoc:127]: Updated the content.

**Pull request merged**